internal/providers/pke/pkeworkflow/create_cluster.go

```diff
@@ -123,6 +123,28 @@ func (w CreateClusterWorkflow) Execute(ctx workflow.Context, input CreateCluster
 		}
 	}
 
+	// Select images (if not specified)
+	{
+		activityInput := SelectImagesActivityInput{ClusterID: input.ClusterID, NodePools: nodePools}
+		err := workflow.ExecuteActivity(ctx, SelectImagesActivityName, activityInput).Get(ctx, &nodePools)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Select volume sizes
+	{
+		activityInput := SelectVolumeSizesActivityInput{
+			AWSActivityInput: awsActivityInput,
+			NodePools:        nodePools,
+		}
+
+		err := workflow.ExecuteActivity(ctx, SelectVolumeSizesActivityName, activityInput).Get(ctx, &nodePools)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Same as above. Although I could think of merging these two activities into one for default values.

Reply:
that might make sense.

Reply:
Fixed in e43d6fd3eedd4d530de6565a1a1150a30e56b9f1.